clusters/build-clusters/common/pod-scaler/serviceaccount.yaml

```diff
@@ -56,3 +56,28 @@ subjects:
 - kind: ServiceAccount
   namespace: ci
   name: pod-scaler
+---
+kind: ClusterRole
```

Comment:
One thing to keep in mind: Since we are not using a cache, we will likely run into delays because of ratelimiting.

/lgtm
/approve

Reply:
We should be able to measure that in webhook metrics I think